src/Controls/src/Core/HandlerImpl/Window/Window.cs

```diff
@@ -17,6 +17,10 @@ public partial class Window
 				// This property is also on the Application Mapper since that's where the attached property exists				
 				[PlatformConfiguration.AndroidSpecific.Application.WindowSoftInputModeAdjustProperty.PropertyName] = MapWindowSoftInputModeAdjust,
 #endif
+
+#if WINDOWS
+				[nameof(ITitledElement.Title)] = MapWindowTitle
+#endif
```

Comment:
Should this be Title or maybe a new TitleBar? Does setting the title actually change any rectangles or things?

Reply:
Same answer as here https://github.com/dotnet/maui/pull/14517#discussion_r1174011497

After this PR is merged then I'll be adding something like `Window.TitleBar` users can specify on `Pages` to swap out the Title Bar.